Tests/UnitTests/Networking/HTTPClientTests.swift

```diff
@@ -27,22 +27,22 @@ class HTTPClientTests: TestCase {
     override func setUp() {
         super.setUp()
 
-        userDefaults = MockUserDefaults()
-        eTagManager = MockETagManager(userDefaults: userDefaults)
-        operationDispatcher = OperationDispatcher()
+        self.userDefaults = MockUserDefaults()
+        self.eTagManager = MockETagManager(userDefaults: self.userDefaults)
+        self.operationDispatcher = OperationDispatcher()
         MockDNSChecker.resetData()
 
-        client = HTTPClient(apiKey: apiKey,
-                            systemInfo: systemInfo,
-                            eTagManager: eTagManager,
-                            dnsChecker: MockDNSChecker.self,
-                            requestTimeout: 3)
+        self.client = HTTPClient(apiKey: self.apiKey,
+                                 systemInfo: self.systemInfo,
+                                 eTagManager: self.eTagManager,
+                                 dnsChecker: MockDNSChecker.self,
+                                 requestTimeout: 3)
     }
 
     override func tearDown() {
-        HTTPStubs.removeAllStubs()
-
         super.tearDown()
+
+        HTTPStubs.removeAllStubs()
```

Comment:
was the ordering here important?

Reply:
Oh no, let me revert this. It's one of the things I tested before I knew what was wrong.